mozilla/blurts-server
# MNTOR-1356: Add Settings E2E test

# References: 
Jira: MNTOR-1356




# Description
The test should check for:
1. the settings page is not broken / is available
2. some mandatory sections of the Settings page are visible


# Screenshot (if applicable)
<img width="778" alt="Screenshot 2023-04-06 at 3 13 06 PM" src="https://user-images.githubusercontent.com/42494162/230503370-2a6a9691-79e1-4b34-9776-cfcea7a3c012.png">


# How to test

CICD: 
* Check the Github actions in this repo or the pipeline in circleCI

Local:
* `npm run e2e`
* `npx playwright show-report` to see report


# Checklist (Definition of Done)
